Index: scripts/hud.py
===================================================================
diff --git a/scripts/hud.py b/scripts-old/hud.py
rename from scripts/hud.py
rename to scripts-old/hud.py
--- a/scripts/hud.py	(revision 383fc92522c54e1cf79cf36e6f3ea4160d2119bf)
+++ b/scripts-old/hud.py	(date 1765643605485)
@@ -137,10 +137,3 @@
         # centraliza dentro do bg_rect
         text_rect = text_surf.get_rect(center=bg_rect.center)
         surface.blit(text_surf, text_rect)
-
-
-# Uso (exemplo mínimo, integrar no loop do jogo):
-# hud = HUD(player=player, pos=(10,10), size=(220,24), font_path='data/fonts/pocketpixel.ttf')
-# no loop:
-#    hud.draw(screen)
-#    pygame.display.flip()
diff --git a/scripts/level_display.py b/scripts-old/level_display.py
rename from scripts/level_display.py
rename to scripts-old/level_display.py
